dapr/clients/grpc/client.py

```diff
@@ -169,8 +174,7 @@ def invoke_service(
         Returns:
             :class:`InvokeServiceResponse` object returned from callee
         """
-        req_data = InvokeServiceRequestData(data, content_type)
-
+        req_data = InvokeServiceRequest(data, content_type)
         http_ext = None
         if http_verb:
             http_ext = self._get_http_extension(http_verb, http_querystring)
```

Comment:
make sure `req_data.is_http()` is true at this point

Reply:
req_data would not have http related data because invoke_service will use argument value directly passed by user code. is_http will be used for callback side. to avoid the confusion, we may need to refactor this part later.